discovery-infra/tests/test_e2e_install.py

```diff
@@ -34,7 +35,7 @@ def test_infra_env_install(self, infra_env, openshift_version):
 
     @JunitTestSuite()
     @pytest.mark.parametrize("is_static_ip", [False, True])
-    @pytest.mark.parametrize("network_type", [NetworkType.OpenShiftSDN, NetworkType.OVNKubernetes])
+    @pytest.mark.parametrize("network_type", [NetworkType.OVNKubernetes])
```

Comment:
wait, what about the previous 2 tests that where running in this job? (SDN+staticIP and SDN-staticIP)